modules/pagination/client-react/containers/PaginationDemo.native.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { translate } from '@restapp/i18n-client-react';
+import { styles as lookStyles, Select } from '@restapp/look-client-react-native';
+import PaginationDemoView from '../components/PaginationDemoView.native';
+import { withDataProvider } from './DataProvider';
+
+@translate('pagination')
+@withDataProvider
+class PaginationDemo extends Component {
+  static propTypes = {
+    t: PropTypes.func,
+    items: PropTypes.object,
+    loadData: PropTypes.func
+  };
+
+  state = {
+    pagination: 'standard',
+    options: [
```

Comment:
Why in the state? Is it supposed to be changed?

Reply:
Fixed